.ci/manywheel/build_common.sh

```diff
@@ -18,12 +18,14 @@ retry () {
     $*  || (sleep 1 && $*) || (sleep 2 && $*) || (sleep 4 && $*) || (sleep 8 && $*)
 }
 
+PLATFORM="manylinux2014_x86_64"
 # TODO move this into the Docker images
 OS_NAME=$(awk -F= '/^NAME/{print $2}' /etc/os-release)
 if [[ "$OS_NAME" == *"CentOS Linux"* ]]; then
     retry yum install -q -y zip openssl
 elif [[ "$OS_NAME" == *"AlmaLinux"* ]]; then
     retry yum install -q -y zip openssl
+    PLATFORM="manylinux_2_28_x86_64"
```

Comment:
nit: this string is set twice here. How about extracting it into a separate variable?

Reply:
We will be moving to only manylinux_2_28_x86_64 builds in couple of week, dropping CentOS 7 support completely. This will be rewritten.